### DISCUSSION TITLE
Importing Spineopt template

### DISCUSSION THREAD

**Discussion by Mastomaki** [spine-tools/SpineOpt.jl#369] [Support]
Running these commands in Julia according to the Getting started guide:

    julia> using SpineOpt

    julia> SpineOpt.import_data("sqlite:////home/jube/learning/spine/tests/projects/getting_started/.spinetoolbox/items/ds_1/ds_1.sqlite", SpineOpt.template(), "Load SpineOpt template")

Leads to error:

    ERROR: UndefVarError: upgrade not defined
    Stacktrace:
    [1] import_data(::String, ::Dict{Symbol,Array{Any,1}}, ::String) at /home/jube/.julia/packages/SpineInterface/EIDQR/src/api.jl:657
    [2] import_data(::String, ::Dict{String,Any}, ::String) at /home/jube/.julia/packages/SpineInterface/EIDQR/src/api.jl:650
    [3] top-level scope at REPL[8]:1

**Comment by manuelma** [comment]
Thanks. Which version of SpineInterface are you using?

**Reply by Mastomaki** [reply]
SpineInterface v0.5.9

**Reply by manuelma** [reply]
It's funny, those line numbers don't match the code in the 0.5.9 release. Are you sure that's the version you're using? I believe this bug is already solved in that version.

**Reply by Mastomaki** [reply]
Hello,

After restarting Julia, my updates only took effect. So now it works!

Best wishes, Jussi

**Reply by manuelma** [reply]
Ok, I'm glad. 

Yeah, by default, Julia doesn't 'reload' a package if the source code changes. To do that, you need to use the [Revise](https://github.com/timholy/Revise.jl) package.
